### ISSUE TITLE
Improve Performance?

### ISSUE THREAD

**Issue by mirisuzanne** [mirisuzanne/blend#6]
Maybe replacing the `array.map` and `array.reduce` functions with basic loops would help? Needs some experimenting…

**Comment by tomasklaen** [comment]
Performance is the reason why I just have to drop this. My sass build times increased from 300ms to 3.7 seconds, which is intolerable :(

**Comment by mirisuzanne** [comment]
Yeah, I also haven't used it much because of the performance issues. Instead, I've been working on how these features will get added to the Sass core. WIP proposal is at https://css.oddbird.net/sass/color-spaces/proposal/
